### PR TITLE
`deno -v` should report typescript version

### LINKED ISSUE #993: deno -v should report typescript version

### PR DESCRIPTION
**Pull request by romandev**
Fixes #993

### CODE REVIEW

File: src/version.rs [hunk 1]
```diff
@@ -4,11 +4,13 @@ use std::ffi::CStr;
 
 // This is the source of truth for the Deno version. Ignore the value in Cargo.toml.
 const DENO_VERSION: &str = "0.1.8";
+const TS_VERSION: &str = "3.1.3";
```
Comment (ry) [inline src/version.rs]: It should be read from elsewhere - this might get out of sync.

Comment (kevinkassimo) [reply src/version.rs]: I don't think we are starting any v8 instance during `deno -v`, thus not able to access `ts.version` through binding.
Without hard-coding, we might have to do some kind of injection on rust side during build.

Comment (kitsonk) [reply src/version.rs]: It will need to be in userland and:

```ts
import { version } from "typescript";
console.log(version);
```

Comment (ry) [reply src/version.rs]: Yea, probably the best way to do this is to send the V8 and Deno versions in the "Start" message to V8, and then print the version from there.

**Comment by romandev** [comment]
I've just addressed your comments but I'm not sure if this is correct (I'm a newbie yet).
If you don't mind, could you please take a look?

File: src/ops.rs [hunk 2]
```diff
@@ -178,6 +179,12 @@ fn op_start(
   let cwd_off =
     builder.create_string(deno_fs::normalize_path(cwd_path.as_ref()).as_ref());
 
+  let v8_version = version::get_v8_version();
+  let v8_version_off = builder.create_string(v8_version);
+
+  let deno_version = version::get_deno_version();
```
Comment (ry) [inline src/ops.rs]: version::DENO_VERSON

File: js/main.ts [hunk 3]
```diff
@@ -59,6 +60,14 @@ export default function denoMain() {
     os.exit(0);
   }
 
+  // handle `--version`
+  if (startResMsg.versionFlag()) {
+    console.log("deno version:", startResMsg.denoVersion());
+    console.log("v8 version:", startResMsg.v8Version());
+    console.log("typescript version:", version);
```
Comment (ry) [inline js/main.ts]: Remove “ version” from the print out

**Pull request merged**